igcollect/ping.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python3
+
+"""igcollect - Latency collection
+Copyright (c) 2020 InnoGames GmbH
+"""
+
+import _thread
+import argparse
+import subprocess
+
+from time import time, sleep
+
+def parse_args():
+    parser = argparse.ArgumentParser(
+        description='Ping colecctor script for Graphana ')
+    parser.add_argument('adress', metavar='URL', type=str, nargs='+',
```

Comment:
typo: `address`

Reply:
also this accepts multiple addresses. so rather name it `addresses`

Reply:
Furthermore it's hosts that are pinged, not necessarily addresses. I assume fping will query DNS.

Reply:
- string is the default type
- Why name it address and then internally use URL ?